samples/storageclass/powerstore-nfs.yaml

```diff
@@ -34,6 +34,20 @@ parameters:
   # Default value: false
   allowRoot: "false"
 
+  # nfsAcls: enables setting permissions on NFS mount directory
+  # This value overrides the NFS ACL (nfsAcls) attribute of corresponding array config in secret, if present
+  # Permissions can be specified in two formats:
+  #   1) Unix mode (NFSv3)
+  #   2) NFSv4 acl (NFSv4)
+  # Allowed values:
+  #   1) Unix mode: valid octal mode number
+  #      Examples: "0777", "777", "0755"
+  #   2) NFSv4 acls: valid NFSv4 acls, seperated by comma
```

Comment:
Please mention that only NFSV4 ACLs are supported

Reply:
done